src/engine/specification.md

```diff
@@ -259,7 +259,7 @@ The payload build process is specified as follows:
 5. Client software **MUST** respond to this method call in the following way:
   * `{status: INVALID_BLOCK_HASH, latestValidHash: null, validationError: errorMessage | null}` if the `blockHash` validation has failed
   * `{status: INVALID_TERMINAL_BLOCK, latestValidHash: null, validationError: errorMessage | null}` if terminal block conditions are not satisfied
-  * `{status: SYNCING, latestValidHash: null, validationError: null}` if the payload extends the canonical chain and requisite data for its validation is missing
+  * `{status: SYNCING, latestValidHash: null, validationError: null}` if requisite data for the payload's acceptance or validation is missing
```

Comment:
I think it worth adding something like "all ancestors of the payload are known and comprise a well-formed chain" to `{status: ACCEPTED ...` statement below. Otherwise, it's not clear that `ACCEPTED` may only be returned when a chain of payload's ancestors is *known*.